tests/parametric/test_otel_span_methods.py

```diff
@@ -620,15 +618,20 @@ def test_otel_span_link_attribute_handling(self, test_agent, test_library):
 
         assert len(link.get("attributes")) == 8
         assert link["attributes"].get("foo") == "bar"
-        assert link["attributes"].get("bools.0") == "true"
-        assert link["attributes"].get("bools.1") == "false"
         assert link["attributes"].get("nested.0") == "1"
         assert link["attributes"].get("nested.1") == "2"
         assert link["attributes"].get("array.0") == "a"
         assert link["attributes"].get("array.1") == "b"
         assert link["attributes"].get("array.2") == "c"
 
-    @missing_feature(context.library == "dotnet", reason="Not implemented")
+        if context.library == "dotnet":
```

Comment:
Could we do a case-insensitive string comparison?

Reply:
Addressed in the latest commit.